packages/@react-aria/grid/src/useGrid.ts

```diff
@@ -175,7 +176,7 @@ export function useGrid<T>(props: GridProps, state: GridState<T, GridCollection<
   );
 
   if (isVirtualized) {
-    gridProps['aria-rowcount'] = state.collection.size;
+    gridProps['aria-rowcount'] = getItemCount(state.collection, (node) => node.type === 'item' || (node.type === 'loader' && node.props.isLoading));
```

Comment:
again, bit unfortunate that the collection's size is not longer reflective of the number of rows since we don't actually want to count the loading sentinel if it isn't actually loading. Shouldn't be that much more expensive hopefully since `getItemCount` does some caching